### DISCUSSION TITLE
Create mssql Mathematical Incident Report

### DISCUSSION THREAD

**Discussion by onmoso** [hasura/graphql-engine#9502] [Question]
Controlled export:
"Error: SyntaxError: Unexpected token 'd', "dbo" is not valid JSON"
mssql tables won't show up.
![image](https://user-images.githubusercontent.com/64882073/225838777-1dcd7e7e-ca19-445c-82bd-9acbd6e2e8c7.png)

**Comment by adas98012** [comment]
@8990679 
Can you add detailed steps to reproduce the problem ? If it is reproducible, please create a Github Issue for better visibility.

**Reply by onmoso** [reply]
OK, thank you.
Github Issue has been created, https://github.com/hasura/graphql-engine/issues/9521
